claudia-lab/model/fruit.js

```diff
@@ -0,0 +1,57 @@
+'use strict';
+
+const uuid = require('node-uuid');
+const createError = require('http-errors');
+const debug = require('debug')('fruit:fruit');
+const storage = require('../lib/storage.js');
+
+const Fruit = module.exports = function(name, texture, color){
+  debug('instantiate fruit');
+  if (!name) throw createError(400, 'expected name');
+  if (!texture) throw createError(400, 'expected texture');
+  if (!color) throw createError(400, 'expected color');
+
+  this.id = uuid.v1(); //generates random id
+  this.name = name;
+  this.texture = texture;
+  this.color = color;
+};
+
+Fruit.createFruit = function(_fruit) {
```

Comment:
@loomnugget nice use of the underscore in your naming convention to differentiate the passed in fruit reference and a newly instantiated fruit - this is a great pattern.